slc/component/matter-core-sdk/energy_tracing.slcc

```diff
@@ -0,0 +1,36 @@
+id: energy_tracing
+package: matter
+label: energy_tracing
+description: >
+  Enables tracing of energy states
+category: Matter|Core
+quality: production
+metadata:
+  sbom:
+    license: "Apache 2.0"
```

Comment:
I understand where this came from but I believe the proper license should be:
license: MSLA, to be verified with the team, I know this is a bit all over the place in our components.